source/wp-content/themes/wporg-parent-2021/sass/block-styles.scss

```diff
@@ -121,3 +121,51 @@
 		}
 	}
 }
+
+// A heading with an arrow and other content.
+.heading-with-arrow {
+	transition: background-color 0.15s linear;
+
+	figure {
+		transition: all 0.15s linear;
+	}
+
+	a {
+		color: var(--wp--preset--color--white) !important;
+		text-underline-offset: 2px;
+		cursor: pointer;
+	}
+
+	&:hover,
+	&:focus-within {
+		transition: background-color 0.15s linear;
+		background-color: rgba(0, 0, 0, 0.1);
+
+		figure {
+			transform: translateX(10px);
+			transition: all 0.15s linear;
+		}
+
+		.heading-with-arrow__row > div > p {
+			text-decoration: underline;
+			text-decoration-thickness: 3px;
+			text-underline-offset: 15px;
+		}
+	}
+
+	.heading-with-arrow__row {
+		cursor: pointer;
+	}
+
+	.heading-with-arrow__list {
```

Comment:
We can switch this list to use [the list of links block style](https://github.com/WordPress/wporg-main-2022/issues/29) once it exists, rather than adding CSS here. (just noting that, no specific change requested now)